### PR TITLE
refactor test partners villages

### PR DESCRIPTION
**Pull request by ayocodingit**
# overview

- refactor test maximum 25 lines

cc: @jabardigitalservice/jds-backend 

## Evidence
- title: refactor test partners villages max 25 lines 
- project: Desa Digital
- participants: @ayocodingit @sandisunandar99 @tukangremot @rindibudiaramdhan

### CODE REVIEW

File: src/modules/villages/village_test.ts [hunk 1]
```diff
@@ -19,27 +19,29 @@ describe('seed data', () => {
...
-              city: expect.any(Object),
-              category: expect.any(Object),
-              location: expect.objectContaining({
-                lat: expect.any(Number),
-                lng: expect.any(Number),
-              }),
-              images: expect.any(Array)
-            })
-          ]),
+          data: expectBodyFindAll,
```
Comment (AldiNFitrah) [inline src/modules/villages/village_test.ts:44]: I prefer to put all the components of a test inside the test itself. Thus, I think `expectBodyFindAll` should be declared inside the `it('test success findAll ...', async () => {`.

ref: https://mtlynch.io/good-developers-bad-tests/#keep-the-reader-in-your-test-function

Comment (AldiNFitrah) [reply src/modules/villages/village_test.ts:44]: by the way, the article is pretty good and can be discussed to be our test convention

Comment (ayocodingit) [reply src/modules/villages/village_test.ts:44]: I make an error if there are more than 25 lines, next discuss test convention

**Review by sandisunandar99 — APPROVED** [review]

**Review by hafizulf — APPROVED** [review]

**Review by rindibudiaramdhan — APPROVED** [review]

**Comment by ayocodingit** [comment]
> So, you move them to constants variables

yes, for reusable too

**Review by firmanJS — APPROVED** [review]

**Pull request merged**